### PR TITLE
gcp sql instance analysis added and tested

### PR DESCRIPTION
**Pull request by Aryan-code-dev**

### CODE REVIEW

File: cartography/intel/gcp/sql.py [hunk 1]
```diff
@@ -39,6 +39,8 @@ def get_sql_instances(sql: Resource, project_id: str, regions: list, common_job_
             if response.get('items', []):
                 for item in response['items']:
                     item['id'] = f"projects/{project_id}/instances/{item['name']}"
+                    for network in item.get('settings', {}).get('ipConfiguration', {}).get('authorizedNetworks',[]):
+                        item['authorizedNetworksList'].append(network['value'])
```
Comment (ganeshjagdale9545) [inline cartography/intel/gcp/sql.py:43]: get `authorizedNetworks ` object like this 
`item[authorizedNetworksList]=item.get('settings', {}).get('ipConfiguration', {}).get('authorizedNetworks',[])`

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
one minor change

**Review by ganeshjagdale9545 — APPROVED** [review]
looks good

**Comment by ganeshjagdale9545** [comment]
@Aryan-code-dev please resolve conflict

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Pull request merged**